recipes-kernel/linux/files/sos_5.4.cfg

```diff
@@ -3,3 +3,43 @@ CONFIG_ACRN_GUEST=y
 CONFIG_DWXPCS=m
 CONFIG_MARVELL_PHY=m
 CONFIG_MARVELL_10G_PHY=m
+
+# OpenStack-related settings
+CONFIG_IP6_NF_IPTABLES=y
+CONFIG_IP6_NF_MATCH_IPV6HEADER=y
+CONFIG_IP6_NF_MATCH_RPFILTER=y
+CONFIG_IP6_NF_FILTER=y
+CONFIG_IP6_NF_TARGET_REJECT=y
+CONFIG_IP6_NF_MANGLE=y
+CONFIG_IP6_NF_RAW=y
+CONFIG_IP6_NF_NAT=y
+CONFIG_IP6_NF_TARGET_MASQUERADE=y
```

Comment:
IP6 configuration is already enabled. Please verify at runtime.
http://git.yoctoproject.org/cgit/cgit.cgi/yocto-kernel-cache/tree/cfg/net/ip6_nf.cfg

Could you update your patch with missing configurations only. Please use *.scc files from yocto-kernel-cache, if configuration is available.

Reply:
@saininav  
CONFIG_IP6_NF_NAT=y
CONFIG_IP6_NF_TARGET_MASQUERADE=y, these two configs need to be added, @peterfang, please modify, thanks